pkg/deploy/assets/aks-development.json

```diff
@@ -612,55 +644,6 @@
                 ],
                 "enableSoftDelete": true
             }
-        },
-        {
-            "type": "Microsoft.Network/virtualNetworkGateways",
-            "name": "aks-vpn",
-            "apiVersion": "2020-08-01",
-            "location": "[resourceGroup().location]",
-            "dependsOn": [
-                "[resourceId('Microsoft.Network/publicIPAddresses', 'aks-vpn-pip')]",
```

Comment:
You can also remove the 'aks-vpn-pip' above this, since its no longer required if there isn't a Gateway using it.